filesystems/nfs/connectathon/runtest.sh

```diff
@@ -109,29 +113,55 @@ function checkServers ()
         echo " ***** Warn: Some servers are offline or unavailable in this domain *****" | tee -a $OUTPUTFILE
         echo " ========== Continue testing with online servers only... ==========" | tee -a $OUTPUTFILE
         echo "" | tee -a $OUTPUTFILE
-        #
-        # Lets actually WARN/ABORTED the test here and investigate,
-        # as we will have incomplete testing results
-        #
-        # report_result $TEST WARN/ABORTED
-        # is_run_byci && rhts-abort -t recipe
-        # We should skip the specific testcase for the offline server instead, a
-        # ton of recipes are aborting. For now, just report a skip until we can
-        # properly work around the issue.
-        rhts-report-result "$TEST" SKIP "$OUTPUTFILE"
-        exit 0
+
+        # Update online servers list
+        servers=${onlineservers}
+
+        if is_run_byci ; then
+            if [[ -z $servers ]]; then
+                # Not found online nfs servers
+                echo "Not found online nfs server from the list, aborting the task" | tee -a $OUTPUTFILE
+                report_result $TEST WARN/ABORTED
+                # Abort the task
+                rstrnt-abort --server $RSTRNT_RECIPE_URL/tasks/$TASKID/status
+                exit 0
+            else
+                # Found online nfs servers
+                echo ""
+                echo "===================================="
+                echo "Testing on online nfs servers:" | tee -a $OUTPUTFILE
+                for S in $servers; do
+                    local server=$(echo $S| cut -f1 -d:)
+                    echo "   |__"${server} | tee -a $OUTPUTFILE
+                done
+                echo ""
+
+                # if found offline servers
+                if [[ ! -z $offlineservers ]]; then
+                    echo ""
+                    echo "================================="
+                    echo "Skipping offline nfs servers:" | tee -a $OUTPUTFILE
+                    for S in $offlineservers; do
+                        echo "   |__"${S} | tee -a $OUTPUTFILE
+                    done
+                    echo ""
+                fi
+            fi
+       else
+          report_result $TEST WARN/ABORTED
```

Comment:
We abort if `is_run_bycib` is false?

Reply:
@emanchado this matches the original implementation for running the test outside of CI